common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -293,49 +300,53 @@ div.video {
           font-size: em(14);
         }
 
-        li {
+        a {
+          @extend %video-button;
           float: left;
-          margin-bottom: 0;
+          background-image: url('../images/vcr.png');
+          background-position: 15px 15px ;
+          background-repeat: no-repeat;
+          border-left: none;
+          box-shadow: 1px 0 0 #555;
+          padding: 0 lh(.75);
+          width: 14px;
 
-          a {
-            @extend %video-button;
-            background-image: url('../images/vcr.png');
-            background-position: 15px 15px ;
-            background-repeat: no-repeat;
-            border-left: none;
-            box-shadow: 1px 0 0 #555;
-            padding: 0 lh(.75);
-            width: 14px;
-
-            &:focus {
-              @extend %ui-depth4;
-              position: relative;
-              outline: $white dotted thin;
-              outline-offset: -2px;
-            }
+          &:focus {
+            @extend %ui-depth4;
+            position: relative;
+            outline: $white dotted thin;
+            outline-offset: -2px;
+          }
 
-            &:empty {
-              height: 46px;
-              background-position: 15px 15px;
-            }
+          &:empty {
+            height: 46px;
+            background-position: 15px 15px;
+          }
 
-            &.play {
-              background-position: 17px -114px;
-            }
+          &.play {
+            background-position: 17px -114px;
+          }
 
-            &.pause {
-              background-position: 16px -50px;
-            }
+          &.pause {
+            background-position: 16px -50px;
           }
 
-          div.vidtime {
-            font-weight: bold;
-            line-height: 46px; //height of play pause buttons
-            -webkit-font-smoothing: antialiased;
-            padding-left: lh(.75);
-            @media (max-width: 1120px) {
-              padding-left: lh(0.5);
-            }
+          &.skip {
+            background-image: none;
+            text-indent: 0;
+            width: initial;
+            white-space: nowrap;
+          }
+        }
+
+        div.vidtime {
+          float: left;
+          font-weight: bold;
+          line-height: 46px; //height of play pause buttons
+          -webkit-font-smoothing: antialiased;
+          padding-left: lh(.75);
+          @media (max-width: 1120px) {
```

Comment:
This seems like an unfortunate requirement due to the fixed pixel based sizing for the video controls. I know that this is only looking at 1 part of the controls, but at some point it would be great to not have to special case this kind of sizing / padding issue in this way. (I may be misinterpreting the role of this styling though.) No action necessary here for this PR.